jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/schemagen/XmlSchemaGenerator.java

```diff
@@ -622,7 +622,7 @@ private void writeTo(Result result, Map<Namespace,String> systemIds) throws IOEx
                     schema._import().namespace(WellKnownNamespace.SWA_URI).schemaLocation("http://ws-i.org/profiles/basic/1.1/swaref.xsd");
                 }
                 if(useMimeNs) {
-                    schema._import().namespace(WellKnownNamespace.XML_MIME_URI).schemaLocation("http://www.w3.org/2005/05/xmlmime");
+                    schema._import().namespace(WellKnownNamespace.XML_MIME_URI).schemaLocation("https://www.w3.org/2005/05/xmlmime");
```

Comment:
This schema location otoh is a url where the schema is found, so there changing https makes sense to me.

Reply:
Thanks, I'll try to test again without this line changed!